xbmc/windowing/windows/WinEventsWin32.cpp

```diff
@@ -167,9 +165,13 @@ static XBMC_keysym *TranslateKey(WPARAM vkey, UINT scancode, XBMC_keysym *keysym
   }
 
   // Attempt to convert the keypress to a UNICODE character
-  GetKeyboardState(keystate);
+  if (!GetKeyboardState(keystate))
```

Comment:
```suggestion
  if (FALSE == GetKeyboardState(keystate))
```

GetKeyboardState retuns type BOOL (not bool) and is more correct compare with same type

Reply:
yes, got confused by MS doc.